cmd/utils/selector.go

```diff
@@ -78,7 +77,7 @@ type SelectorItem struct {
 }
 
 //AskForOptionsOkteto given some options ask the user to select one
-func AskForOptionsOkteto(ctx context.Context, options []SelectorItem, label, selectedTpl string) (string, bool, error) {
+func AskForOptionsOkteto(ctx context.Context, options []SelectorItem, label, selectedTpl string, initialPosition int) (string, bool, error) {
```

Comment:
Are we using `ctx`? I think we could remove it since the code that was using it has been moved to another file

Reply:
this was changed some commits after :) i think u where viewing a specific commit filter